components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/dao/SAMLSSOServiceProviderDAO.java

```diff
@@ -232,15 +240,27 @@ public boolean addServiceProvider(SAMLSSOServiceProviderDO serviceProviderDO) th
             throw new IdentityException("Issuer cannot be found in the provided arguments.");
         }
 
+        // If an issuer qualifier value is specified, it is appended to the end of the issuer value
```

Comment:
`// If an issuer qualifier value is specified, it is appended to the end of the issuer value.`

Reply:
Addressed in https://github.com/wso2/carbon-identity-framework/pull/3100